tests/cypress/tests/02-search/01-search.spec.js

```diff
@@ -6,105 +6,98 @@
 /// <reference types="cypress" />
 
 import { squad, tags } from '../../config'
-import { cliHelper } from '../../scripts/cliHelper'
+import { deployment, namespace } from '../../../common-lib/resources'
+import {
+  cliHelper,
+  generateNewMultiResourceState,
+  resetNewMultiResourceState,
+} from '../../scripts/cliHelper'
 import { searchPage, searchBar } from '../../views/search'
 
+// rework
```

Comment:
Delete comment or explain it better.

Reply:
Deleted this comment. This was more of a note that I forgot to remove.